unattended_installer/install_functions/checks.sh

```diff
@@ -166,6 +166,27 @@ function checks_arguments() {
 
 }
 
+# Checks if the --retry-connrefused is available in curl
+function check_curlVersion() {
+
+    if [ "${sys_type}" == "yum" ]; then
+        if ! yum list installed 2>/dev/null | grep -q curl; then
+            eval "yum install curl -y ${debug}"
+        fi
+    elif [ "${sys_type}" == "apt-get" ]; then
+        if ! apt list --installed 2>/dev/null | grep -q "${dep}"; then
```

Comment:
Same

Reply:
This check is now reworked. Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/55b08088e52d97c87b15a608dadf48cf9b9cd70e.